### PR TITLE
adding mongodbnotify to community

### PR DESCRIPTION
**Pull request by bsbishop**
This uses an Atlas-MongoDB cluster (M0-free) to allow for notifications to be sent from anywhere but I mainly built it to allow Tidbyt to display notifications from Home Assistant.
mongodb_notify

### CODE REVIEW

**Comment by bsbishop** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bsbishop** [comment]
Any movement on this? I'll be happy to hop on a call and explain / demonstrate, if it helps.

File: apps/mongodbnotify/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+id: mongodb-notify
+name: MongoDB Notify
+summary: Message notifications
+desc: Uses MongoDB for notifications. Can be used with Home Assistant to get Notifications to Tidbyt.
```
Comment (matslina) [inline apps/mongodbnotify/manifest.yaml:5]: Could you please update the app description a bit? Would be good if it explained that users will likely have to go read the README here in the community repo to get this set up.

Comment (bsbishop) [reply apps/mongodbnotify/manifest.yaml]: Hi, @matslina,
I've updated the desc as you requested.

**Pull request merged**